web_widget_mail_send_odoo/static/src/js/web_widget_mail_send_odoo.js

```diff
@@ -104,4 +103,67 @@ openerp.web_widget_mail_send_odoo = function(instance) {
 
     instance.web.form.widgets.add('email_mailto', 'instance.web.form.FieldEmail')
     instance.web.form.widgets.add('email', 'instance.web.form.FieldEmailIntern')
+
+    instance.web_kanban.FieldEmailInternKanban = instance.web_kanban.AbstractField.extend({
+        init: function(parent, field, $node) {
+            this.parent = parent;
+            this._super.apply(this, arguments);
+            this.format_descriptor = _.extend({}, this.field, {'widget': this.$node.attr('widget')});
+        },
+        is_syntax_valid: function(){
+            var self = this;
+            var value = self.get('value');
+            if (value) {
+                try {
+                    var parsed_email = instance.mail.ChatterUtils.parse_email(value);
+                    if(!parsed_email[1])
+                        return false;
+                    return true;
+                } catch(e) {
+                    return false;
+                }
+            }
+            return true;
+        },
+        on_clicked: function() {
```

Comment:
shouldn't you cancel the event so that the browser doesn't run its default action?